kubernetes/kube.libsonnet

```diff
@@ -703,7 +703,7 @@
 
   LimitRange(name, namespace): $._Object('v1', 'LimitRange', name, namespace=namespace),
 
-  PodDisruptionBudget(name, namespace, app=name): $._Object('policy/v1beta1', 'PodDisruptionBudget', name, namespace=namespace) {
+  PodDisruptionBudget(name, namespace, app=name): $._Object('policy/v1', 'PodDisruptionBudget', name, namespace=namespace) {
```

Comment:
Add to PR description?

This is needed for the 1.25 upgrade, if I'm reading things correctly, and should be available starting at 1.21.

Is this function used in any 1.14 clusters, like the old ops cluster?

Reply:
The conditionality of this argument is actually more difficult to fix than it looks initially. 

I thought about making a conditional or variable argument to the function and default to `v1beta1`, with an override for modern deployments. But I am thinking to do the opposite now. Default to `policy/v1` and for deployments that go to ops clusters they will need to be updated to override to `policy/v1beta1`

Reply:
I pushed an update for jsonnet-libs which I think works for us. But it makes the tradeoff to move forward and requires us to modify backward for ops cluster(ops.us-west-2). I’ll identify which apps will need updates and follow up on those.

Reply:
This seems overly-aggressive to me, given that this upgrade isn't happening for at least a quarter. Why not just wait until the ops cluster is retired, and save yourself some work?

Reply:
Ok, I was not thinking about the timeline, I can take out the PodDisruptionBudget change